recipes/simfil/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+
+
+required_conan_version = ">=1.62.0"
+
+
+class SimfilRecipe(ConanFile):
+    name = "simfil"
+    description = "simfil is a C++ 17 library and a language for querying structured map feature data. The library provides an efficient in-memory storage pool for map data, optimized for the simfil query language, along with a query interpreter to query the actual data."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Klebert-Engineering/simfil"
+    license = "BSD 3-Clause"
+    topics = ["query language"]
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_json": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_json": True,
+    }
+
+    exports_sources = "include/*", "src/*"
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "10",
+            "clang": "10",
+            "apple-clang": "14",
+        }
+
+    def validate(self):
+        check_min_cppstd(self, 20)
+
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warning(f"{self.name} recipe lacks information about the {self.settings.compiler} compiler support.")
+        else:
+            if Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.name} requires Concepts support. The current compiler {self.settings.compiler} {self.settings.compiler.version} does not support it.")
+
+    def build_requirements(self):
+        self.build_requires("cmake/3.28.1")
+
+    def requirements(self):
+        self.requires("sfl/1.2.4", transitive_headers=True)
+        self.requires("fmt/10.2.1", transitive_headers=True)
+        self.requires("bitsery/5.2.3", transitive_headers=True)
+        if self.options.with_json:
+            self.requires("nlohmann_json/3.11.2")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["SIMFIL_CONAN"] = True
+        tc.cache_variables["SIMFIL_SHARED"] = bool(self.options.get_safe("shared"))
+        tc.cache_variables["SIMFIL_WITH_REPL"] = False
+        tc.cache_variables["SIMFIL_WITH_COVERAGE"] = False
+        tc.cache_variables["SIMFIL_WITH_TESTS"] = False
+        tc.cache_variables["SIMFIL_WITH_EXAMPLES"] = False
+        tc.cache_variables["SIMFIL_WITH_MODEL_JSON"] = bool(self.options.with_json)
```

Comment:
```suggestion
        tc.cache_variables["SIMFIL_WITH_MODEL_JSON"] = self.options.with_json
```

The CMakeToolchain converts to boolean automatically. CMake executable will receive "True", but it's totally capable to understand that string.